From 6974372fbbea4279e1e4f5c24425581208f5553a Mon Sep 17 00:00:00 2001
From: Nikita Popov <nikita.ppv@gmail.com>
Date: Mon, 18 Oct 2021 12:45:26 +0200
Subject: [PATCH] Fix bug #81510

Make the used arrays larger, because the previous sizes were not
slow enough on some hardware.
---
 Zend/tests/bug74093.phpt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Zend/tests/bug74093.phpt b/Zend/tests/bug74093.phpt
index 7f20285805bf..c5fc9da8cbbf 100644
--- a/Zend/tests/bug74093.phpt
+++ b/Zend/tests/bug74093.phpt
@@ -12,8 +12,8 @@ max_execution_time=1
 hard_timeout=1
 --FILE--
 <?php
-$a1 = range(1, 1000000);
-$a2 = range(100000, 1999999);
+$a1 = range(1, 2000000);
+$a2 = range(100000, 2999999);
 array_intersect($a1, $a2);
 ?>
 --EXPECTF--
